src/game.cpp

```diff
@@ -185,6 +189,50 @@ bool Game::update() {
     return true;
 }
 
+void Game::open_menu() {
+    Menu menu(settings_.width, settings_.height);
+    while (window_.isOpen()) {
+        sf::Event event;
+        while (window_.pollEvent(event)) {
+            switch (event.type) {
+                case sf::Event::KeyPressed:
+                    switch (event.key.code) {
+                        case sf::Keyboard::Up:
+                            menu.MoveUp();
+                            break;
+                        case sf::Keyboard::Down:
+                            menu.MoveDown();
+                            break;
+                        case sf::Keyboard::Q:
+                            settings_.running = false;
+                            return;
+                        case sf::Keyboard::Enter:
+                            switch (menu.getPressedItem()) {
+                                case Menu::Options::PLAY:
+                                    return;
+                                case Menu::Options::INFO:
+                                    menu.open_info(settings_.width,
+                                                   settings_.height, window_);
+                                    break;
+                                case Menu::Options::QUIT:
+                                    settings_.running = false;
+                                    return;
+                                default:
+                                    break;
+                            }
+                        default:
+                            break;
+                    }
+                default:
+                    break;
+            }
+        }
+        window_.clear(sf::Color(1, 3, 50));
```

Comment:
ten kolor daj do jakiejś stałej i nazwij (nie wiadomo co to jest za kolor na pierwszy rzut oka)